docs/examples/defaultlabelprovider/no-translations.js

```diff
@@ -0,0 +1,60 @@
+// @flow strict
+import { type Node, useState } from 'react';
```

Comment:
Note that we no longer need to import `React` — we can import only what's needed in the file, same as in all our other files